KarhooUISDK.xcodeproj/project.pbxproj

```diff
@@ -4532,7 +4532,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				MARKETING_VERSION = 1.7.2;
+				MARKETING_VERSION = 1.8.0;
```

Comment:
the build number is usually bumped as well

Reply:
done